thumbor/config.py

```diff
@@ -131,6 +131,7 @@
 Config.define('ALLOW_OLD_URLS', True, 'Indicates if encrypted (old style) URLs should be allowed', 'Security')
 Config.define('ENABLE_ETAGS', True, 'Enables automatically generated etags', 'HTTP')
 Config.define('MAX_ID_LENGTH', 32, 'Set maximum id length for images when stored', 'Storage')
+Config.define('GC_INTERVAL', 60, 'Set garbage collection interval in seconds', 'Performance')
```

Comment:
I went ahead and made the change. Personally I think it should default to on but if you think it's best to default to off then that's what we'll do.